RELEASE_SUPPORT.md

```diff
@@ -16,6 +16,7 @@ The table below outlines the *eProsima Fast DDS* minor releases and their suppor
 
 |Version branch|Release Date|EOL Date|
 |--------------|------------|--------|
+|v2.10.x (not yet released)|March 2023|May 2024 [^*]|
```

Comment:
Should we extend `v2.6.x` EOL date? Or we can extend the date when we modify this file again doing the release.

Reply:
I think we can leave that decision to the moment when we do the release of 2.10.0